packages/node_modules/@ciscospark/test-helper-test-users/src/index.js

```diff
@@ -12,8 +12,8 @@ if (typeof Promise === 'undefined') {
 /* eslint-env mocha */
 /* eslint camelcase: [0] */
 
-const _ = require('lodash');
 const assert = require('assert');
+const _ = require('lodash');
```

Comment:
This may complain in linting. I believe it checks for alphabetizing.

Reply:
This file is no longer changed